src/pipeline_manager.py

```diff
@@ -63,6 +63,10 @@ def train(pipeline_name, validation_size, logger, params):
     meta_train = meta[meta['is_train'] == 1]
     meta_train_split, meta_valid_split = train_valid_split(meta_train, validation_size, random_state=SEED)
 
+    if dev_mode:
```

Comment:
@apyskir I think it is better to have dev mode data_size in pipeline_configs . That way if you are testing something (like augmentation bug) for which you may want to choose a different number. Then it is better to change in just one place

Reply:
@jakubczakon Done